x230/README.md

```diff
@@ -64,13 +64,15 @@ supported anymore, once you're running coreboot (You'd have to manually
 flash back your backup images first, see later chapters).
 
 This updates the BIOS _and_ Embedded Controller (EC) firmware. The EC
-is not updated anymore, when running coreboot. The latest EC version is 1.14
-and that's unlikely to change.
+is not updated anymore, when running coreboot. The latest EC version is 1.15
+which is included with the latest official BIOS release 2.77 from 04 Oct 2019.
+
```

Comment:
please don't say "latest" bios release because that'll again become outdated eventually. "since official BIOS release 2.77" would be good I guess.